src/util/xdrquery/XDRQueryEval.cpp

```diff
@@ -472,6 +472,12 @@ FieldList::getFieldNames() const
     return names;
 }
 
+inline std::string
+format_as(const NullField&)
```

Comment:
What is this change for?

Reply:
without this it won't compile with the new version of the fmt library
They fixed a bug in the way `fmt::format` works, and we're supposed to provide custom formatters for custom types

Reply:
Interesting, do we still need the old formatter defined on line 11?

Reply:
yes done, didn't see it